NEWS.md

```diff
@@ -0,0 +1,131 @@
+# Cosmos-SDK v0.39.0 Release Notes
```

Comment:
Why don't you just call this file `RELEASE_NOTES.md`?

Reply:
The filename was inspired by a naming convention that is wide spread across open source projects https://www.gnu.org/prep/standards/standards.html#NEWS-File

If you feel particularly strongly in favour of renaming this to `RELEASE_NOTES.md` (or strongly against keeping it as it is), then I'll happily rename it. @clevinson @ethanfrey any thoughts on this?

Please do note that I'm planning to copy and paste the contents in the release I'll publish in Github too anyway :)

Reply:
I'm fine either way. Don't have a strong preference. If we are making use of a NEWS file a standard, i would suggest documenting it somewhere in our release procedure, README.md, or CONTRIBUTING.md, so that its clear to users where to find this info.

Reply:
I've renamed `NEWS.md` to `RELEASE_NOTES.md`. A self-explanatory name is worth 1000s additional lines in CONTRIBUTING.md. Thanks @clevinson!